src/tests/conformance/cmd/squelch.rs

```diff
@@ -15,9 +15,7 @@
 //!     <- mtPROPOSE_LEDGER (validator1)
 //!     <- mtPROPOSE_LEDGER (validator2)
 //!     -> mtSQUELCH (validator1)
-//!     <-
 //!     <- mtPROPOSE_LEDGER (validator2)
-//!     <-
```

Comment:
How would you improve this example? Or we just remove this example from here and that's it 😀

Reply:
Done